### ISSUE TITLE
v3.0.0-dev.1 - Inspector throwing error: Invalid value: Not in inclusive range 0..1: 49

### ISSUE THREAD

**Issue by SimonHausdorf** [isar/isar#434]
Just checking if this is a known issue in the dev version.

**Steps to Reproduce**
I have a simple model like this:
```dart
@Collection()
@JsonSerializable()
class TestModel {
  int? id;
  String coin;
  String symbol;
  double price;

  TestModel({
    this.id,
    this.coin = '',
    this.symbol = '',
    this.price = 0,
  });

  factory TestModel(Map<String, dynamic> json) =>
      _$TestModelFromJson(json);
}
```

Then I use `isar.put(TestModel(coin: 'test', symbol: 'test', price: 3.2));`. Looking at the web inspector, the content isn't loaded and I see the following error in the console: `error: Invalid value: Not in inclusive range 0..1: 49`

**Version**
 - Platform: Mac, iOS Simulator
 - Flutter version: 3.0.2
 - Isar version: 3.0.0-dev.1
 - Isar Generator version: 3.0.0-dev.1

**Comment by Jictyvoo** [comment]
Have the same issue here on Linux + Firefox 101

**Comment by leisim** [comment]
Should be fixed in 3.0.0-dev.3
